src/engine/DebugOverlays.EntityLabel.cs

```diff
@@ -22,6 +23,49 @@ private bool ShowEntityLabels
         }
     }
 
+    private void UpdateLabelColour(IEntity iEntity, Spatial entity, Label label)
```

Comment:
Why not access the Spatial node from IEntity's `EntityNode`? That way there don't have to be these 2 similar params.